app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    all_data = command.split(" ")
```

Comment:
There is no need to specify that the delimiter is space in `split()` it is a default value.